zebra-scan/Cargo.toml

```
@@ -19,6 +19,11 @@ name = "scanner-grpc-server"
 path = "src/bin/rpc_server.rs"
 required-features = ["proptest-impl"]
 
+[[bin]] # Bin to run the Scanner tool
+name = "zebra-scanner"
+path = "src/bin/scanner/main.rs"
+required-features = ["shielded-scan"]
+
```

Could we compile the binary by default?

```suggestion
[[bin]] # Bin to run the Scanner tool
name = "zebra-scanner"
path = "src/bin/scanner/main.rs"

```

---

https://github.com/ZcashFoundation/zebra/pull/8608/commits/3fcb0b1e176311283c8422afcaa3e64be2a4a9ef